test/addons-napi/test_instanceof/test.js

```diff
@@ -74,7 +74,8 @@ if (typeof Symbol !== 'undefined' && 'hasInstance' in Symbol &&
   compareToNative(x, MySubClass);
   compareToNative(y, MySubClass);
   compareToNative(x, MyClass);
-  compareToNative(y, MyClass);
+  // TODO: https://github.com/nodejs/abi-stable-node/issues/236
```

Comment:
Maybe they could be moved to nodejs/node-chakracore/issues. I think sometime soon we will stop using the abi-stable-node fork, now that all the code has been merged from it. But then we'll review all the remaining open issues there.